creator/studies/mutations.py

```diff
@@ -205,6 +218,15 @@ def mutate(self, info, input, workflows=None):
             )
 
         attributes = sanitize_fields(input)
+
+        # Check short_code is provided and unique
```

Comment:
If you make this required in the input object, you don't need this check. Graphene will do it for you

Reply:
it will check it's provided right? but not uniqueness?